data/json/items/generic.json

```
@@ -2338,6 +2338,7 @@
     "price_postapoc": 10,
     "material": [ "cotton" ],
     "weight": "514 g",
+	"container": "pillowcase",
```

You need to lint your JSON (this suggestion does it), and you also don't have down-filled pillows spawning in them.
You can use [this website](http://dev.narc.ro/cataclysm/format.html) to lint JSON. Simply paste an entire file in, format it, then paste it back into the file.
```suggestion
    "container": "pillowcase",
```